test/core/origin_core/helpers/utils.js

```diff
@@ -105,6 +107,54 @@ Utils.prototype = {
              initialValidators,
              initialStakes,
         );
+    },
+
+    assertCommitMetaBlock: function(
+         events,
+        height,
+        kernelHash,
+        transitionHash,
+        requiredWeight,
+        verifiedWeight
+    ) {
+
+
+        assert(
+             events.MetaBlockCommitted !== undefined,
+             `Commit meta-block event not emitted`
+        );
+        assert.equal(
```

Comment:
Is it possible to use `strictEqual()`? Or if it is `BN`, then `BN::eq()`?
(For all asserts)

Reply:
I tried `strictEqual` doesn't work.